plugins/woocommerce/src/Blocks/Templates/ComingSoonSocialLinksTemplate.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Automattic\WooCommerce\Blocks\Templates;
+
+/**
+ * ComingSoonSocialLinksTemplate class.
+ *
+ * @internal
+ */
+class ComingSoonSocialLinksTemplate extends AbstractTemplatePart {
+
+	/**
+	 * The slug of the template.
+	 *
+	 * @var string
+	 */
+	const SLUG = 'coming-soon-social-links';
+
+	/**
+	 * The template part area where the template part belongs.
+	 *
+	 * @var string
+	 */
+	public $template_area = 'uncategorized';
+
+	/**
+	 * Initialization method.
+	 */
+	public function init() {}
+
+	/**
+	 * Returns the title of the template.
+	 *
+	 * @return string
+	 */
+	public function get_template_title() {
+		return _x( 'Page: Coming soon social links', 'Template name', 'woocommerce' );
```

Comment:
I'm unsure where this appears, but do you know if `Page: ` is used at all? I don't think "page" would be the accurate description here, but I may miss something 🤔

Reply:
You're right! It's not a page. I created the template based on the existing coming soon template and didn't remove it. It's a template part. 

Removed the "Page:" prefix in 3611f1a148ecb11dae4054901c67162f968ba099. I was expecting it to be shown in the Patterns section, but it looks like it's not. However, it's required because the template part class extends AbstractTemplate, which requires a title and description.